docs(writing-a-loader): added webpack-default description (#3468)
webpack/webpack.js.org: src/content/contribute/writing-a-loader.md

Last but not least, if you've already created a separate repository and package for your loader, you could [`npm link`](https://docs.npmjs.com/cli/link) it to the project in which you'd like to test it out.

> T> Use [`webpack-default`](https://github.com/webpack-contrib/webpack-defaults) to setup your webpack loader project. It will generate a webpack's standard project for your loader

Reviewer: ```suggestion
T> Use [`webpack-default`](https://github.com/webpack-contrib/webpack-defaults) to setup your webpack loader project. It will generate a webpack standard project for your loader.
```

> T> Use [`webpack-defaults`](https://github.com/webpack-contrib/webpack-defaults) to setup your webpack loader project. It will generate a webpack standard project for your loader.

Reviewer 2: ```suggestion
T> Use [`webpack-defaults`](https://github.com/webpack-contrib/webpack-defaults) to generate a boilerplate for your loader.
```

Reviewer 2: Sounds more concise, wdyt?

Author: I guess `webpack standard boilerplate` might fit more right?

Reviewer 2: `standard webpack boilerplate for your loader` sounds more correct.

Author: Cool, I will change that.

Reviewer 2: Appreciate it!

> T> Use [`webpack-defaults`](https://github.com/webpack-contrib/webpack-defaults) to setup your webpack loader project. It will generate a webpack standard project for your loader.

Reviewer 3: ```suggestion
T> You can use [`webpack-defaults` package](https://github.com/webpack-contrib/webpack-defaults) to generate boilerplate code necessary to start writing your loader.
```
